appengine/standard/migration/taskqueue/pull-counter/README.md

```diff
@@ -0,0 +1,77 @@
+# App Engine Task Pull Queue Migration Sample
+
+This sample replaces the
+[App Engine Tasks Queue Pull Counter sample](../../../taskqueue/pull-counter)
+that used the `taskqueue` library
+available only in the App Engine Standard for Python 2.7 runtime.
+
+The sample uses a
+[Pub/Sub topic with a pull queue](https://cloud.google.com/pubsub/docs/pull)
+as recommended in
+[Migrating pull queues to Pub/Sub](https://cloud.google.com/appengine/docs/standard/python/taskqueue/pull/migrating-pull-queues)
+to perform the same functions as the earlier, Task Queue sample.
+
+The application has three functions:
+
+* Viewing the home page will display a form to specify a task name, and add
+  one more request for it. It will also show all requested tasks and their counts.
+
+* Submitting the form on the home page will queue a task request.
+
+* Requesting the page at `/_ah/start` will begin processing the queued tasks.
+  This will run without responding to the request until the request is
+  cancelled, or the request times out, which can be a significant amount
+  of time.
+
+## Setup
```

Comment:
are there ways to do either all of these via `gcloud` commands or all of them via the UI? It seems a little funky to do two console and a CLI

Reply:
soft preference would be all CLI for ease of maintenance and because copy pasting is great

Reply:
The Firestore/Datastore mode management can only be done via the console, so far as I can tell from the documentation.

I could go either way in enabling the API, but I personally find it clearer to use the console for this.